test/oneof.test.js

```diff
@@ -401,3 +401,105 @@ test('oneOf object with field of type string with format or null', (t) => {
     prop: toStringify
   }), `{"prop":"${toStringify.toISOString()}"}`)
 })
+
+test('one array item do not match oneOf types', (t) => {
```

Comment:
I think this test title should be positive, not negative.

Reply:
It really should be positive, my 🇺🇸 that tricked me